From 7a82ba7332ecf0561a596efbdc27df748dc891ab Mon Sep 17 00:00:00 2001
From: Eric Koskinen <eric.koskinen@stevens.edu>
Date: Thu, 18 Jun 2026 13:01:02 -0400
Subject: [PATCH 3/6] options fix

---
 src/analysis/analyze.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/analysis/analyze.ml b/src/analysis/analyze.ml
index 5f2f016..36c409f 100644
--- a/src/analysis/analyze.ml
+++ b/src/analysis/analyze.ml
@@ -229,5 +229,5 @@ let verify_of_block e genv _ blks vars pre post : bool option * bool option =
   let embedding = generate_embedding_map vars in
   let [@warning "-8"] spec , [m1;m2] = Spec_generator.compile_blocks_to_spec genv blks embedding pre post in
   let cond = (fst @@ Spec_generator.exp_to_smt_exp e 1 Spec_generator.variable_ctr_list) in
-  Servois2.Verify.verify spec m1 m2 cond,
+  Servois2.Verify.verify ~options:!Util.servois2_verify_option spec m1 m2 cond,
   Servois2.Verify.verify ~options:{(!Util.servois2_verify_option) with ncom = true} spec m1 m2 (EUop(Not, cond))
-- 
2.50.1 (Apple Git-155)

